src/sprite_clone.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * This file is part of EasyRPG Player.
+ *
+ * EasyRPG Player is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * EasyRPG Player is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with EasyRPG Player. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef EP_DRAWABLE_CLONE_H
+#define EP_DRAWABLE_CLONE_H
+
+// Headers
+#include "sprite.h"
+
+/**
+ * DrawableClone class.
+ */
+class SpriteClone : public Sprite {
+public:
+	SpriteClone(Sprite* drawable, int nx, int ny, int nz);
+	~SpriteClone() override;
```

Comment:
found a bug: In the sprite.h file change ``~Sprite() override;`` to ``virtual ~Sprite() override;`` otherwise the objects are not correctly deleted.